Sources/Common/Extensions/StringExtension.swift

```diff
@@ -362,7 +364,11 @@ public extension String {
     // MARK: Host name validation
 
     var isValidHost: Bool {
-        return isValidHostname || isValidIpHost
+        return (isValidHostname || isValidIpHost) && !isMathFormula
+    }
+
+    var isMathFormula: Bool {
```

Comment:
Should this be internal or private? Not big deal just wondering if we need to expose it.

Reply:
Good catch. It should be private.